src/app/clusters/network-commissioning/network-commissioning.cpp

```diff
@@ -61,11 +61,13 @@ namespace app {
 namespace clusters {
 namespace NetworkCommissioning {
 
-constexpr uint8_t kMaxNetworkIDLen       = 32;
-constexpr uint8_t kMaxThreadDatasetLen   = 254; // As defined in Thread spec.
-constexpr uint8_t kMaxWiFiSSIDLen        = 32;
-constexpr uint8_t kMaxWiFiCredentialsLen = 64;
-constexpr uint8_t kMaxNetworks           = 4;
+constexpr uint8_t kMaxNetworkIDLen          = 32;
+constexpr uint8_t kMaxThreadDatasetLen      = 254; // As defined in Thread spec.
+constexpr uint8_t kMaxWiFiSSIDLen           = 32;
+constexpr uint8_t kMaxWiFiCredentialsLen    = 64;
+constexpr uint8_t kMaxNetworks              = 4;
+constexpr uint8_t kMaxScannedThreadNetworks = 4;
+constexpr uint8_t kMaxScannedWiFiNetworks   = 4;
```

Comment:
Are we using kMaxScannedWifiNetwworks? Should this replace kMaxNetworks?

Reply:
No, it is not used yet, I added it by the way of adding `kMaxScannedThreadNetworks`, as I believe after adding scan support for WiFi array similar to the `sThreadScanResults` should appear for WiFi e.g. `sWiFiScanResults[kMaxScannedWiFiNetworks]`. For now I will remove it. And it is not the same as `kMaxNetworks`, because it describes maximum number of networks allowed to add and `kMaxScanned...` are maximum number of networks that can be scanned in the device surroundings and send back in the response packet.